### ISSUE TITLE
⌨ Investigate found Automated WCAG conformance issues

### ISSUE THREAD

**Issue by RM-1978** [GCTC-NTGC/design-gc-digital-talent#21]
## ⌨ Accessibility conformance verification 

An Automated Review of the platform using SiteImprove revealed the WCAG 2 conformance errors in the screenshot below. 

## 📸 Screenshot

![Image](https://user-images.githubusercontent.com/101275838/261427707-dd0349e0-5f8c-4b96-8e63-f37f8e139b43.png)

### WCAG 2.1 Criteria 

List of  WCAG A and AA criteria to be confirmed. New issues will be created depending on the location of the errors. 

## 🦋 Expected Behaviour

- All level A and AA issues found by SiteImprove are confirmed and verified as real (e.g. not false positives). 
- Well defined GitHub issues are created to remedy the verified conformance issues.  

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] No WCAG conformance errors
- [ ] Separate GitHub Tracker Issues created where applicable

**Comment by RM-1978** [comment]
@gobyrne and @esizer Here are the results of the SiteImporve tests. We will need to sit down to discuss these resulta with Lauren and the design team as most of the issues seem to be on the design side :(


## 📸 Screenshot

![Image](https://github.com/GCTC-NTGC/design-gc-digital-talent/assets/101275838/daf5a773-9db1-40e1-8595-e881146b2629)

**Comment by substrae** [comment]
As Lee mentioned, the AAA stuff should be broken away for a later conversation. We don't have the resources to process them right now. Meeting AAA contrast requirements are impossible with our current palette, so we'd end up having to rebrand the whole product. Altering something as fundamental as line-height requires a rework of every single component in our library. Increasing line height too much is arguably bad for legibility, and our line heights also vary depending on the element for layout purposes, meaning there are likely false positives here.

The two we should prioritize fixing, from what I can tell, are the ARIA attribute error and the form element contrast problems. We'll need to know specifics about each:
* Which ARIA attribute(s) are unsupported or prohibited?
* Which form elements aren't meeting color contrast (looks like there are only 3 instances)

**Comment by substrae** [comment]
I also noticed that in the original report summary image there are 16 A level errors/warnings, but they don't seem to be present in the more detailed report screenshot. Any idea about what they were or where they went?

**Comment by RM-1978** [comment]
@substrae Thanks for your comments!

Upon further investigation, SiteImprove has confirmed that the overall automated score for  GC Talent (https://talent.canada.ca/en/ ) score is: **94.4** / 100. 

The score to be AA compliant from an automated testing perspective for this site is **96.7** / 100. The nature of the issues they found were based on conformance to WCAG 2.1.  

They include: 
- AA issues on site – Color contrast does not meet minimum requirement in some places. 
- Design issue on forms (3 occurrences on 1 page)
- ARIA issues on site – ARIA attribute unsupported or prohibited – development issue (55 occurrences across 4 pages)
- Content missing after heading & overuse of italics are Siteimprove specific 

The good news is that they also confirmed that, upon rescan ( on Dec. 6, 2023), no level A issues were detected on GC Talent. 	

I have also invited them in to demo their tool to better understand a bit more about it and how they arrived at these results :)

**Comment by substrae** [comment]
Great, that sounds a lot more accurate than the original results!

**Comment by gobyrne** [comment]
Closing. We can re-open if we get actionable details on the failings. (aka we can see the report)
